cabal.project

```diff
@@ -65,7 +65,10 @@ constraints:
   -- https://github.com/ndmitchell/hlint/issues/1376
   hlint +ghc-lib,
   ghc-lib-parser-ex -auto,
-  stylish-haskell +ghc-lib
+  stylish-haskell +ghc-lib,
+  -- implicit-hie 0.1.3.0 introduced an unexpected behavioral change.
+  -- https://github.com/Avi-D-coder/implicit-hie/issues/50
+  implicit-hie < 0.1.3
```

Comment:
See https://github.com/haskell/haskell-language-server/pull/3317#discussion_r1050818694, this applies here, too. @michaelpj what do you think? Fix in a follow-up PR? Must not be forgotten for #3317

Reply:
Yeah, good point. I was thinking it was okay to be temporary, but indeed it can mess up our ability to release. Better to put a fake dependency in `ghcide` if we have to.

Reply:
added implicit-hie to ghcide.